libraries/standard/http/utest/http_utest.c

```diff
@@ -9,6 +9,96 @@
+      HTTP_TEST_REQUEST_PATH_LEN + SPACE_CHARACTER_LEN +            \
+      HTTP_PROTOCOL_VERSION_LEN + HTTP_HEADER_LINE_SEPARATOR_LEN +  \
+      HTTP_USER_AGENT_FIELD_LEN + HTTP_HEADER_FIELD_SEPARATOR_LEN + \
+      HTTP_USER_AGENT_VALUE_LEN + HTTP_HEADER_LINE_SEPARATOR_LEN +  \
+      HTTP_HOST_FIELD_LEN + HTTP_HEADER_FIELD_SEPARATOR_LEN +       \
+      HTTP_TEST_HOST_VALUE_LEN + HTTP_HEADER_LINE_SEPARATOR_LEN +   \
+      HTTP_CONNECTION_FIELD_LEN + HTTP_HEADER_FIELD_SEPARATOR_LEN + \
+      HTTP_HEADER_LINE_SEPARATOR_LEN +                              \
+      HTTP_HEADER_LINE_SEPARATOR_LEN )
+
+/* Add HTTP_CONNECTION_KEEP_ALIVE_VALUE_LEN to account for longest possible
+ * length of template header. */
+#define HTTP_TEST_MAX_INITIALIZED_HEADER_LEN \
+    ( HTTP_TEST_PREFIX_HEADER_LEN + HTTP_CONNECTION_KEEP_ALIVE_VALUE_LEN )
+
+/* Add 1 because snprintf(...) writes a null byte at the end. */
+#define HTTP_TEST_INITIALIZED_HEADER_BUFFER_LEN \
+    ( HTTP_TEST_MAX_INITIALIZED_HEADER_LEN + 1 )
+
+/* Template HTTP header fields and values. */
+#define HTTP_TEST_HEADER_FIELD               "Authorization"
+#define HTTP_TEST_HEADER_FIELD_LEN           ( sizeof( HTTP_TEST_HEADER_FIELD ) - 1 )
+#define HTTP_TEST_HEADER_VALUE               "None"
+#define HTTP_TEST_HEADER_VALUE_LEN           ( sizeof( HTTP_TEST_HEADER_VALUE ) - 1 )
+/* Template for first line of HTTP header. */
+#define HTTP_TEST_HEADER_REQUEST_LINE        "GET / HTTP/1.1 \r\n"
+#define HTTP_TEST_HEADER_REQUEST_LINE_LEN    ( sizeof( HTTP_TEST_HEADER_REQUEST_LINE ) - 1 )
+#define HTTP_REQUEST_HEADERS_INITIALIZER     { 0 }
+/* Template for snprintf(...) strings. */
+#define HTTP_TEST_SINGLE_HEADER_FORMAT       "%s%s: %s\r\n\r\n"
+#define HTTP_TEST_DOUBLE_HEADER_FORMAT       "%s%s: %s\r\n%s: %s\r\n\r\n"
+
+/* Length of the following template HTTP header.
+ *   <HTTP_TEST_HEADER_REQUEST_LINE> \r\n
+ *   <HTTP_TEST_HEADER_FIELD>: <HTTP_TEST_HEADER_VALUE> \r\n
+ *   \r\n
+ * This is used to initialize the expectedHeader string. */
+#define HTTP_TEST_SINGLE_HEADER_LEN       \
+    ( HTTP_TEST_HEADER_REQUEST_LINE_LEN + \
+      HTTP_TEST_HEADER_FIELD_LEN +        \
+      HTTP_HEADER_FIELD_SEPARATOR_LEN +   \
+      HTTP_TEST_HEADER_VALUE_LEN +        \
+      HTTP_HEADER_LINE_SEPARATOR_LEN +    \
+      HTTP_HEADER_LINE_SEPARATOR_LEN )
+
+/* The longest possible header used for these unit tests. */
+#define HTTP_TEST_DOUBLE_HEADER_LEN     \
+    ( HTTP_TEST_SINGLE_HEADER_LEN +     \
+      HTTP_TEST_HEADER_FIELD_LEN +      \
+      HTTP_HEADER_FIELD_SEPARATOR_LEN + \
+      HTTP_TEST_HEADER_VALUE_LEN +      \
+      HTTP_HEADER_LINE_SEPARATOR_LEN )
+
+#define HTTP_TEST_DOUBLE_HEADER_BUFFER_LEN \
+    ( HTTP_TEST_DOUBLE_HEADER_LEN + 1 )
+
+#define HTTP_TEST_BUFFER_LEN    ( 1024U )
+
+static uint8_t httpBuffer[ HTTP_TEST_BUFFER_LEN ] = { 0 };
+
 /* ============================ UNITY FIXTURES ============================== */
 void setUp( void )
```

Comment:
I just noticed this is un-commented like the rest, could you comment for consistency?